trajectory_executor/config/trajectory_executor.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2018-2019 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# Integer: Spin rate for the subscribers, timers, and publishers in the Trajectory Executor
+# Units: Hz
+spin_rate: 10
+
+# Integer: Publish rate for the outbound trajectories to the control plugins.
+# Should be matched to timestep size in TrajectoryPlan inputs
+# Units: Hz
+trajectory_publish_rate: 10
+
+# String: Name of default control plugin. Should match field in TrajectoryPlan message
+# Due to lack of existing plugin discovery mechanism this is the only control plugin
+# which will be available to TrajectoryExecutor
+# Units: N/a
+default_control_plugin: pure_pursuit
+
+# String: Full path to default control plugin's trajectory input topic
+# Units: N/a
+default_control_plugin_topic: /carma/guidance/control_plugins/pure_pursuit/trajectory
```

Comment:
My memory is pretty vague. I forgot if we need each topic for each plugin or just one topic for all of them.

Reply:
I think the plan is one topic per each plugin, but for now it only supports 1 plugin since we don't have an implementation for our new plugin manager component.